EIPS/eip-5727.md

```diff
@@ -0,0 +1,626 @@
+---
+eip: 5727
```

Comment:
Can I clarify a question: this EIP heavily assume there is a concept of `slotId` in addition to `tokenId`, right?

Reply:
Yes, you can refer to EIP-3525 for a more detailed explanation of slotId.